tests/override_test_sqlalchemy_store.py

```diff
@@ -1,29 +1,155 @@
+import time
+from typing import List, Union
+
 import pytest
+from mlflow import entities
+from mlflow.entities import (
+    Param,
+)
 from mlflow.exceptions import MlflowException
 from mlflow.store.tracking.sqlalchemy_store import SqlAlchemyStore
+from mlflow.utils.validation import (
+    MAX_TAG_VAL_LENGTH,
+)
+
+
+def get_current_time_millis():
+    """
+    Returns the time in milliseconds since the epoch as an integer number.
+    """
+    return int(time.time() * 1000)
+
+
+def _create_experiments(store: SqlAlchemyStore, names) -> Union[str, List]:
+    if isinstance(names, (list, tuple)):
+        ids = []
+        for name in names:
+            # Sleep to ensure each experiment has a unique creation_time for
+            # deterministic experiment search results
+            time.sleep(0.001)
+            ids.append(store.create_experiment(name=name))
+        return ids
+
+    time.sleep(0.001)
+    return store.create_experiment(name=names)
+
+
+def _run_factory(store: SqlAlchemyStore, config=None):
+    if not config:
+        config = _get_run_configs()
+    if not config.get("experiment_id", None):
+        config["experiment_id"] = _create_experiments(store, "test exp")
+
+    return store.create_run(**config)
+
+
+def _get_run_configs(experiment_id=None, tags=None, start_time=None):
+    return {
+        "experiment_id": experiment_id,
+        "user_id": "Anderson",
+        "start_time": get_current_time_millis() if start_time is None else start_time,
+        "tags": tags,
+        "run_name": "name",
+    }
+
+
+def _verify_logged(store, run_id, metrics, params, tags):
```

Comment:
This just looks like a copy past from the original Python code.
I do not understand why we do this.